EasyPost/Client.cs

```diff
@@ -108,5 +112,35 @@ public Client(ClientConfiguration clientConfiguration)
 
             _dotNetVersion = dotNetVersion;
         }
+
+        /// <summary>
+        /// Return the request time from the client object.
+        /// </summary>
+        public int getConnectionTimeout(){
+            return _defaultConnectTimeoutMilliseconds;
```

Comment:
We should probably have an intermediate variable called something like `connectionTimeoutMilliseconds` which is either the user's input or the fallback of the default. If we only ever return the default, this getter will never pick up on the user's input.

Reply:
Thanks for the feedback, I will rename those two defaults variables